### ISSUE TITLE
Animation cannot be disabled on mobile devices

### ISSUE THREAD

**Issue by parfilov** [elementor/elementor#7355]
## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
There was stated that the issue was resolved in release 2.5 at [this post](https://github.com/elementor/elementor/issues/1639). The animation on mobile still shows. Looking on Android smartphone.

## Steps to reproduce
I use these settings
<img width="337" alt="animations" src="https://user-images.githubusercontent.com/12493645/53947325-be904c80-40ef-11e9-8eb2-1a7a9ffcbce4.png">

## Isolating the problem
- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.10 (Unix)
	MySQL version: (LTD BeGet) v5.7.21-20
	PHP Version: 7.2.5
	PHP Max Input Vars: 5000
	PHP Max Post Size: 1047527424
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1
	Site URL: http://test.wplovers.pw
	Home URL: http://test.wplovers.pw
	WP Multisite: No
	Max Upload Size: 999 MB
	Memory limit: 268435456
	Permalink Structure: /%category%/%postname%/
	Language: ru-RU
	Timezone: 6
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Seventeen
	Version: 2.1
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: ru_RU
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 YaBrowser/19.3.0.2489 Yowser/2.5 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.5.3
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.5
		Author: Elementor.com


== Log ==
	: 
Log: showing 19 of 192019-01-21 13:05:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.1',
  'to' => '2.4.2',
)]
2019-01-21 13:17:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.3.1',
  'to' => '2.4.0-beta4',
)]
2019-01-21 20:10:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.2',
  'to' => '2.4.3',
)]
2019-01-21 20:10:43 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.2',
  'to' => '2.4.3',
)]
2019-01-21 20:11:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.0-beta4',
  'to' => '2.4.0',
)]
2019-01-25 22:03:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.3',
  'to' => '2.4.4',
)]
2019-01-25 22:05:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.0',
  'to' => '2.4.2',
)]
2019-02-01 15:36:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-02-01 15:36:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.2',
  'to' => '2.4.3',
)]
2019-02-13 18:01:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.5',
  'to' => '2.4.6',
)]
2019-02-13 18:02:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.3',
  'to' => '2.4.4',
)]
2019-02-19 01:25:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.6',
  'to' => '2.4.7',
)]
2019-02-19 01:25:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-02-21 11:55:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.7',
  'to' => '2.5.0-beta1',
)]
2019-02-22 02:53:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0-beta1',
  'to' => '2.5.0-beta2',
)]
2019-02-26 20:56:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0-beta2',
  'to' => '2.5.0-beta3',
)]
2019-03-04 19:57:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0-beta3',
  'to' => '2.5.0',
)]
2019-03-05 00:27:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0',
  'to' => '2.5.1',
)]
2019-03-07 15:41:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.1',
  'to' => '2.5.3',
)]

JS: showing 4 of 4JS: 2019-01-25 16:13:58 [error X 3][http://test.wplovers.pw/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.4:2:6883] Cannot read property \'select2\' of undefined 
JS: 2019-01-28 20:30:38 [error X 1][http://test.wplovers.pw/wp-admin/js/editor.min.js?ver=5.0.3:1:694] Cannot read property \'style\' of undefined 
JS: 2019-01-28 20:30:38 [error X 1][http://test.wplovers.pw/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.4:2:209315] Cannot read property \'on\' of null 
JS: 2019-02-21 06:01:57 [error X 13][http://test.wplovers.pw/wp-content/plugins/elementor/assets/lib/nouislider/nouislider.min.js?ver=13.0.0:2:22846] noUiSlider (13.0.0): Slider was already initialized. 

PHP: showing 4 of 4PHP: 2019-01-29 02:27:12 [notice X 65][/home/a/alexpa4p/test.wplovers.pw/public_html/wp-content/plugins/elementor/core/files/css/base.php::703] Undefined index: _id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-02-13 18:13:02 [error X 1][/home/a/alexpa4p/test.wplovers.pw/public_html/wp-content/plugins/elementor/includes/editor.php::711] Allowed memory size of 268435456 bytes exhausted (tried to allocate 58195968 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-02-26 23:14:19 [error X 2][/home/a/alexpa4p/test.wplovers.pw/public_html/wp-content/plugins/elementor/includes/editor.php::711] Allowed memory size of 268435456 bytes exhausted (tried to allocate 59801600 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-02-27 22:57:26 [error X 2][/home/a/alexpa4p/test.wplovers.pw/public_html/wp-content/plugins/elementor/includes/editor.php::711] Allowed memory size of 268435456 bytes exhausted (tried to allocate 59813888 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

```
</details>

**Comment by booth1** [comment]
I have the same issue. I am using animation on a Popup and the animation doesn't load for ages which makes things look broken, and although like you it shows no animation on Tablet and Mobile, it definitely still happens. Hopefully a fix out there!

**Comment by DmitriyDSWEB** [comment]
+1 need add function NONE when select mobile or tablet

**Comment by bainternet** [comment]
Fixed in 2.5.4

thanks.
